package.json

```diff
@@ -43,7 +43,6 @@
     "configurationModule": "lib/imperative.js"
   },
   "peerDependencies": {
-    "@zowe/cli": "7.0.0-next.202103152050",
     "@zowe/imperative": "5.0.0-next.202104071400"
```

Comment:
Should the Imperative peer dependency version have a `^` in front?

Reply:
If I remember semver correctly, when you have a "beta string" following the version number, the ^ will upgrade to every future version: 5.1.0, 6.0.0, 7.2.5, etc. It makes the ^ pretty useless for beta versions. With the current specification, npm will only use that specific beta version - a nuisance when the beta version changes, but at least it is safe. When we return to a normal GA version number, I agree that we want the ^.

Reply:
Makes sense to leave it as is then, we'll have to update this version number when `@next` becomes `@zowe-v2-lts` anyway.